test/hack/resource/pkg/resourcetypes/instanceprofile.go

```diff
@@ -47,6 +49,9 @@ func (ip *InstanceProfile) GetExpired(ctx context.Context, expirationTime time.T
 
 	errs := make([]error, len(out.InstanceProfiles))
 	for i := range out.InstanceProfiles {
+		if ip.Global() && !strings.Contains(lo.FromPtr(out.InstanceProfiles[i].Arn), lo.Must(config.LoadDefaultConfig(ctx)).Region) {
```

Comment:
What is this for?

Reply:
The cleanup will discover instancesProfiles that are intended for resources outside of sweepers region. This is to scope the clean up to the region the sweeper was intended.